ember-app/app/controllers/issue/create.js

```diff
@@ -17,6 +24,15 @@ var IssuesCreateController = Ember.Controller.extend({
   isValid: function () {
     return this.get("model.title");
   }.property("model.title"),
+  issueBodyDirty: function(){
+    if(this.get('previousRepo.issue_template')){
+      return this.get('model.body') !== this.get('previousRepo.issue_template');
+    }
+    return !Ember.isEmpty(this.get('model.body'));
```

Comment:
You might flip this check to be first - if the body is empty, we should never consider it dirty.